pkg/kubectl/cmd/set/set_serviceaccount_test.go

```diff
@@ -351,7 +344,7 @@ func TestSetServiceAccountRemote(t *testing.T) {
 						return nil, fmt.Errorf("unexpected request")
 					}
 				}),
-				VersionedAPIPath: path.Join(input.apiPrefix, testapi.Default.GroupVersion().String()),
+				VersionedAPIPath: versionedAPIPath(input.path),
```

Comment:
I'm a little concerned that we have to specify a full path, and that then we're trying to guess from the path in this method. Is there a better way to handle/construct the path?

Reply:
Removed the "VersionedAPIPath" string, reducing the size of the input path. Also makes the path consistent with other tests.